src/materials_dict_master.py

```diff
@@ -0,0 +1,8 @@
+materials_dict_master = { 'UO2': {'rho':10.5, 'k': 2.5, 'c': 332},
```

Comment:
unbalanced parens, and it seems like you want this to be a list, given the way you are using it

Reply:
I think this is all we need for the material properties function. Hard code the values into the nested dictionary and have it return the dictionary. I don't think we need to pass an input to the function either.